pkg/rm/two_phase.go

```diff
@@ -37,37 +37,33 @@ var (
 	typError                    = reflect.Zero(reflect.TypeOf((*error)(nil)).Elem()).Type()
 	typContext                  = reflect.Zero(reflect.TypeOf((*context.Context)(nil)).Elem()).Type()
 	typBool                     = reflect.Zero(reflect.TypeOf((*bool)(nil)).Elem()).Type()
-	typBusinessContextInterface = reflect.Zero(reflect.TypeOf((*tm.BusinessActionContext)(nil))).Type()
+	TypBusinessContextInterface = reflect.Zero(reflect.TypeOf((*tm.BusinessActionContext)(nil))).Type()
 )
 
 type TwoPhaseInterface interface {
-	Prepare(ctx context.Context, params ...interface{}) (bool, error)
+	Prepare(ctx context.Context, params interface{}) (bool, error)
```

Comment:
The return type is more appropriate with an interface
```suggestion
	Prepare(ctx context.Context, params interface{}) (interface{}, error)
```

Reply:
> The return type is more appropriate with an interface

we can disguss it in a new issue